src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/InputRegressionJavaMultilineStringsAst.java

```diff
@@ -0,0 +1,20 @@
+//non-compiled with javac: Compilable with Java13
+package com.puppycrawl.tools.checkstyle.grammar;
+
+public class InputRegressionJavaMultilineStringsAst {
+
+    {
+        String doubleQuotes = """ "" """;
+        String singleQuote = """ " """;
+        String empty = """""";
+        String lineFeed = """
+                """;
```

Comment:
Please add another example with text between `"""` and on different line.

Reply:
done

Reply:
@strkkk Why did this change add new lines to everything in this file? Now you have no examples of ML string where everything is on 1 line.

I just wanted a new example similar to the original `lineFeed` but adding text on a new line.

Reply:
Because this fails the build.
Multiline string should have a line break otherwise it wont compile. 

I will also update UTs

Reply:
Thanks for clarification.